package.json

```diff
@@ -99,13 +99,9 @@
     "xlsx": "^0.18.5"
   },
   "jest": {
-    "transform": {
-      "\\.js$": [
-        "rollup-jest"
-      ]
-    },
```

Comment:
Do we need this? It leads to source mapping issues, with test error messages pointing to the wrong line. It also becomes impossible to step through the tests using a debugger (at least in JetBrains). The pointer stops at random places.